src/pages/index.tsx

```diff
@@ -213,10 +217,36 @@ const ApplicationRow = ({ application }: ApplicationRowProps) => {
   );
 };
 
+type TitledCardProps = {
+  title: string;
+  items: JSX.Element[] | undefined;
+};
+const TitledCard = ({ title, items }: TitledCardProps) => {
+  return (
+    <Card className="w-2/3 mx-auto p-6">
+      <span>
+        <Text h3 b>
+          {title}
+        </Text>
+      </span>
+      <div className="grid grid-auto-rows gap-4">
+        {items ? (
```

Comment:
Interesting idea but I would honestly just use `children` here since it's easier to follow